### PR TITLE
updates ui when images dragged into collection

### PR DESCRIPTION
**Pull request by tsop14**
The two methods (dragging and clicking on add button) both use 'addCollectionToImage' function, so checks for new collection and updates ui when images are dragged in to a collection.  

adding-images

Changes 'add a new sub-collection' icon. #perfectmatch ⚡️

screen shot 2016-01-22 at 12 02 35

### CODE REVIEW

File: kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
                 title="Add a new sub-collection"
                 ng:if="ctrl.editing"
                 ng:click="$parent.active = !$parent.active">
-            <gr-icon>add_box</gr-icon>
+            <gr-icon>create_new_folder</gr-icon>
```
Comment (theefer) [inline kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html]: What does that look like out of interest?

File: kahuna/public/js/services/api/collections-api.js [hunk 2]
```diff
@@ -134,8 +137,8 @@ collectionsApi.factory('collections',
         addChildTo,
         isDeletable,
         removeFromList,
-        addImageIdsToCollection,
-        addImagesToCollection,
+        addCollectionUsingImageIds,
+        addCollectionUsingImageResources,
```
Comment (theefer) [inline kahuna/public/js/services/api/collections-api.js]: Should it be add *to* collection? Else I wrongly read it as adding (creating) a new collection.

File: kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html (hunk 1 above)
Comment (theefer) [inline kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html]: Ignore me, found the screenshot, doh.

Comment (theefer) [inline kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html]: If we're going for collection = folder, maybe we should replace the icon on the collection tab?

**Comment by theefer** [comment]
Nice :+1:

File: kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html (hunk 1 above)
Comment (tsop14) [inline kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html]: @jamesgorrie do you think of them as folders?

**Pull request merged**